src/interfaces/stepper-interfaces/PeopleAndRoles/org-person-interface.ts

```diff
@@ -16,3 +16,15 @@ export interface OrgPersonIF {
   deliveryAddress?: AddressIF
   action?: ActionTypes
 }
+
+export interface getOrgPersonsIF {
+  appointmentDate: string,
+  cessationDate: string,
+  deliveryAddress: AddressIF,
+  mailingAddress: AddressIF,
+  officer: {
```

Comment:
No `officer.id` here? This will need to be fixed (future sprint when we handle director changes).

Reply:
Sounds good 👍  Currently no ID returned from the backend. Put it in, optional for now.